features/wijzigingen.feature

```diff
@@ -0,0 +1,100 @@
+
+    Scenario: Gebruiker heeft een volgindicatie op een persoon en er is geen wijziging doorgevoerd in de registratie bij de persoon
+        Gegeven gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999993483" en geen einddatum
+        En sinds 5 maart 2021 is er in de BRP registratie geen wijziging doorgevoerd in de gegevens van de persoon "999993483"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999993483"
+
+    Scenario: Wijzigingen vóór het zetten van de volgindicatie worden niet gemeld
+        Gegeven op 4 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999990457"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999990457" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999990457"
+
+    Scenario: Gebruiker zet een volgindicatie met een einddatum in de toekomst
+        Gegeven gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992855" met einddatum "2021-03-16"
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992855"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "999992855"
+
+    Scenario: Gebruiker zet een volgindicatie met een einddatum in het verleden of vandaag
+        Gegeven gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992806" met einddatum "2021-03-15"
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992806"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992806"
+    
+    Scenario: Gebruiker heropent de volgindicatie
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999994220" met einddatum "2021-03-01"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999994220" en geen einddatum
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999994220"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "999994220"
+
+    Scenario: Gebruiker heropent de volgindicatie en wijzigingen vóór het heropenen worden niet gemeld
```

Comment:
Als ik dit goed begrijp, dan wordt bij het heropenen van een volgindicatie niet alleen het einddatum aangepast, maar ook een niet zichtbare begindatum? En dat dit ook bepalend is voor de bevraging resultaat? Ik weet niet of de consumer dit gaat begrijpen

Reply:
als ik vandaag aangeef dat ik wijzigingen wil doorkrijgen, verwacht ik toch geen wijzigingen van gisteren?

Reply:
Het is mij niet duidelijk wat met heropenen wordt bedoeld. Zoals je het nu aangeeft, concludeer ik dat heropenen betekent een **nieuwe** volgindicatie plaatsen, waarbij ook de niet zichtbare begindatum wordt gewijzigd. Ik ging er van uit dat heropenen betekent alleen de einddatum wijzigen.

Reply:
alleen de einddatum wijzigen geeft onvoorspelbare resultaten, want voor het "heropenen" zijn er mogelijk geen wijzigingen bijgehouden. Het is dan voor de gebruiker niet voorspelbaar of deze de relevante gegevens krijgt.

De update API is bedoeld om een lokale kopie bij te houden. Bij het starten van volgen begint de gebruiker met het ophalen van de actuele gegevens van de persoon. Bij heropenen moet dat ook en kan de gebruiker dus niet vertrouwen op het ophalen van /wijzigingen sinds de vorige periode dat de persoon gevolgd werd.